.github/workflows/proof-pypi.yml

```diff
@@ -3,6 +3,10 @@ on:
   push:
     branches:
       - "master"
+    pull_request:
```

Comment:
As it been discussed with @g4s8, we keep this part without any changes. But if you would ask my opinion, I think we should make this change with all the proofs which have a positive build status